src/gui/tristateaction.h

```diff
@@ -46,6 +46,9 @@ class TriStateAction : public QWidgetAction
     // should use this function instead of QAction::setChecked(bool)
     void setCheckState(Qt::CheckState checkState);
 
+    // When set to 'false' toggling the checkbox will only emit a signal on
+    // QAction::toggled, NOT on QAction::triggered. When placed in a QMenu, this
+    // prevents the menu from being closed.
     void setCloseOnTriggered(bool enabled);
```

Comment:
Maybe better to rename it to `setCloseOnInteraction` to match the new behavior?
Also applies to `TriStateWidget::m_closeOnTriggered` and `TriStateWidget::setCloseOnTriggered()`.

Reply:
Done.